tensorflow_datasets/translate/en_tam_parallel_text.py

```diff
@@ -0,0 +1,97 @@
+# # coding=utf-8

```

Comment:
Please check [tf code style](https://www.tensorflow.org/community/contribute/code_style) and pylint this code.

Reply:
Please review the changes now

Reply:
@us